specification/specs/query_dsl/Operator.ts

```diff
@@ -1,4 +1,6 @@
 enum Operator {
 	and = 0,
-	or = 1
+	or = 1,
+  AND = 2,
```

Comment:
Do we need both casings? I think only caps suffices?

Reply:
Both are being used in the flight recorder :S